### PR TITLE
Added test coverage dependency

### PR DESCRIPTION
**Pull request by meretelutz**

### CODE REVIEW

**Comment by meretelutz** [comment]
Can look at line coverage in the terminal with:
poetry run pytest --cov=quanteda

Can look at branch coverage in the terminal with:
`poetry run pytest --cov-branch --cov=quanteda`

**Comment by meretelutz** [comment]
@jbarns14 you might want to take another look at your tests. When I run the code above your coverage is 0% so I think your tests might not be running right.

**Comment by dorisyycai** [comment]
That's a good idea! I wonder if I need to update this line of code when I run it? I'm receiving below error.
image

**Comment by meretelutz** [comment]
I think you might have to install pytest-cov first before running that line. Try entering 
`poetry add --dev pytest-cov`
before the other lines

**Comment by JohnShiuMK** [comment]
Thanks @meretelutz for adding this
Jake's test coverage is 0% just because we haven't merge his branch yet 
I have just migrate the changes from his branch to this branch

Now it work:
Screenshot 2024-01-17 at 8 24 58 PM

**Comment by JohnShiuMK** [comment]
Btw, I am not sure if you guys have this problem
I need to run `poetry install` to reflect any changes from the pyproject.toml, but that would lead to error unless I delete the poetry.lock file

Is it a better idea not to commit the poetry.lock?

**Comment by JohnShiuMK** [comment]
Also, @meretelutz, do we need the testing.ipynb in the root directory?

File: testing.ipynb
Comment (JohnShiuMK) [inline testing.ipynb:1]: Do we need this file for something?

**Review by JohnShiuMK — COMMENTED** [review]
Other than the testing.ipynb, I think we're ready to approve this PR

**Comment by meretelutz** [comment]
Shoot, sorry I accidentally added that! Definitely not needed!

**Review by JohnShiuMK — APPROVED** [review]
Thanks @meretelutz :)

**Pull request merged**